WordPress/src/main/java/org/wordpress/android/ui/reader/utils/ReaderImageScanner.java

```diff
@@ -81,6 +77,32 @@ public ReaderImageList getImageList(int minImageWidth) {
     }
 
     /*
+     * returns true if there at least `minImageCount` images in the post content that are at
+     * least `minImageWidth` in size
+     */
+    public boolean hasUsableImageCount(int minImageCount, int minImageWidth) {
```

Comment:
See my comment below for a possible improvement. You can basically disregard this comment since that one already covers this.